erdjs-snippets/lending-pool/lendingPool.spec.ts

```diff
@@ -59,29 +62,60 @@ describe("lending snippet", async function () {
     });
 
 
-    it("setup", async function () {
+    it("Create Liquidity Pool", async function () {
         session.expectLongInteraction(this);
 
         await session.syncUsers([whale]);
 
         let token = await session.loadToken("tokenABC");
         let lendAddress = await session.loadAddress("contractAddress");
-        let interactor = await createInteractor(session, lendAddress);
+        let interactor = await createLendingInteractor(session, lendAddress);
 
         // Setup Liquidity pool
         let returnCodeSetup = await interactor.addLiquidityPool(whale, token.identifier, 0, 40000000, 1000000000, 800000000, 100000000, 700000000);
         assert.isTrue(returnCodeSetup.isSuccess());
 
+    });
+
+    it("Issue Lending Token", async function () {
+        session.expectLongInteraction(this);
+
+        await session.syncUsers([whale]);
+
+        let token = await session.loadToken("tokenABC");
+        let lendAddress = await session.loadAddress("contractAddress");
+        let interactor = await createLendingInteractor(session, lendAddress);
+
         // Issue Lend Tokens
-        returnCodeSetup = await interactor.issueLend(whale, token.identifier);
+        let returnCodeSetup = await interactor.issueLend(whale, token.identifier);
```

Comment:
Since the setup has been split etc., `returnCodeSetup` can be renamed to simply `returnCode`.